### PR TITLE
Som devel 20

### PR DESCRIPTION
**Pull request by sorgom**

### CODE REVIEW

File: application/components/LCR/README.md
Comment (sombeam) [inline application/components/LCR/README.md]: something went wrong here

File: SIL4.md
Comment (sombeam) [inline SIL4.md:32]: finish before merge

**Review by sombeam — CHANGES_REQUESTED** [review]

File: application/components/LCR/README.md
Comment (sorgom) [reply application/components/LCR/README.md]: done

File: SIL4.md
Comment (sorgom) [reply SIL4.md:32]: on hold ...

**Review by sombeam — APPROVED** [review]

**Pull request merged**
